package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Node.js implementation for the MPRIS D-Bus Interface Specification to create a mediaplayer service",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "./node_modules/.bin/jasmine"
```

Comment:
You can just write `jasmine`, NPM will expand it to `node_modules/.bin/jasmine` automatically.